### PR TITLE
Feature/39522

### PR DESCRIPTION
**Pull request by UrvashiChaudhary**
Create a method initBodyParser in Server.ts file
This will implement the body parser for the express app.
Please refer the URL for documentation(https://www.npmjs.com/package/body-parser#examples)
Call this method from the bootstrap method of the same class.

Create folder routes inside libs

Create a file notFoundRoute.ts
Create a middleware which will return error 'Not Found'.

Create a file errorHandler.ts
Create a middleware which returns an error in the following format:
{
error: "Not Found",
message: "error",
status: 500,
timestamp: "2019-01-08T17:38:21.929Z"
}

### CODE REVIEW

File: src/Server.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
-// create a class and define methods according to the ticket#39521
-import * as express from "express";
+// create a class and define methods according to the ticket#39522
+import * as express from 'express';
+import * as bodyParser from 'body-parser';
+import { notFoundHandler, errorHandler } from './libs/routes';
+import notFoundRoute from "./libs/routes/notFoundRoute";
 class Server {
```
Comment (adilsuccessive) [inline src/Server.ts:6]: initBodyParser implementation is missing
notFoundRoute and errorHandler routes are also missing.

**Review by adilsuccessive — CHANGES_REQUESTED** [review]

File: src/Server.ts (hunk 1 above)
Comment (UrvashiChaudhary) [reply src/Server.ts:6]: done

File: src/Server.ts [hunk 2]
```diff
@@ -14,8 +17,13 @@ const { app }=this;
 app.get('/health-check',( req, res, next ) => {
 res.send("I am fine");
 });
+app.use(notFoundHandler);
+app.use(errorHandler);
 return this;
 }
+initBodyParser() {
+    this.app.use(bodyParser.json({ type: 'application/*+json' }))
```
Comment (adilsuccessive) [inline src/Server.ts]: its better to use most recommended way to use body-parser with Express.
from the doc
Express route-specific
This example demonstrates adding body parsers specifically to the routes that need them. In general, this is the most recommended way to use body-parser with Express.

**Review by adilsuccessive — CHANGES_REQUESTED** [review]

File: src/Server.ts (hunk 2 above)
Comment (UrvashiChaudhary) [reply src/Server.ts]: done

**Pull request merged**